src/LucidFrame/Console/ConsoleTable.php

```diff
@@ -54,10 +56,10 @@ public function addHeader($content = '')
 
     /**
      * Adds the headers for the columns
-     * @param  array  Array of header cell content
-     * @return object LucidFrame\Console\ConsoleTable
+     * @param array $content Array of header cell content
+     * @return $this
      */
-    public function setHeaders(array $content)
+    public function addHeaders(array $content)
```

Comment:
I think it is better we keep the name `setHeaders`. "add" means we can call it multiply times to add an item into a stack whereas "set" means it overwrites the previously added item.

So, I think we could update the function description comment at line 58.